src/target.cpp

```diff
@@ -721,7 +724,7 @@ void get_target_triple(Buf *triple, const ZigTarget *target) {
             ZigLLVMGetSubArchTypeName(target->sub_arch),
             ZigLLVMGetVendorTypeName(target->vendor),
             ZigLLVMGetOSTypeName(get_llvm_os_type(target->os)),
-            ZigLLVMGetEnvironmentTypeName(target->abi));
+            target_abi_name(target->abi));
```

Comment:
WASISysroot doesn't exist in LLVM so I had to shim it in here.

Reply:
Looking at it closer, I defined the enum in ZigLLVM space. Maybe I should move this transform back into ZigLLVMGetEnvironmentTypeName?